### PR TITLE
@N-Kannan #8557

### PR DESCRIPTION
**Pull request by N-Kannan**
I have updated the Document

For the issue #8557
A member of the role db_owner can execute sp_updatestats for Azure SQL Database. #8557

 Please reader the documentation And Approve the pull request. As per i did Google i god the Detail and updated

### CODE REVIEW

**Comment by Court72** [comment]
@markingmyname 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/relational-databases/system-stored-procedures/sp-updatestats-transact-sql.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ When restoring a database to [!INCLUDE[sssql22-md](../../includes/sssql22-md.md)
 
 ## Permissions
 
-To run `sp_updatestats`, the user must be the owner of the database (the `dbo`, not just member of the role `db_owner`) or to be member of the sysadmin fixed server role.
+To run sp_updatestats, the user must be the owner of the database (the dbo) or a member of the db_owner fixed database role. For Azure SQL Database, a user with the db_owner role can execute sp_updatestats. Users with the sysadmin fixed server role can also execute the stored procedure.
```
Comment (rwestMSFT) [inline docs/relational-databases/system-stored-procedures/sp-updatestats-transact-sql.md]:
```suggestion
On SQL Server, you must be the owner of the database (**dbo**), or a member of the **db_owner** fixed database role or **sysadmin** fixed server role.

For Azure SQL Database, you must be a member of the **db_owner** fixed database role.
```

**Comment by N-Kannan** [comment]
@markingmyname can you complete the pull request. If it have complete information

**Comment by rwestMSFT** [comment]
@N-Kannan It can take days or weeks to respond to contributions, especially with a major conference around the corner that may involve a document freeze. We appreciate your patience.

**Comment by rwestMSFT** [comment]
This change is being pulled into a private PR, to update the article in other respects. This PR will be closed, but you will still get credit for your change.

**Pull request closed without merging**